docker-compose-cli.yaml

```diff
@@ -58,8 +49,6 @@ services:
       driver: none 
     volumes:
       - postgres:/var/lib/postgresql/data
-    ports:
-      - "5432:5432"
```

Comment:
Hide all exposed ports except BE UI (`3010`) to reduce the number of possible conflicts with other running apps and containers.